src/views/repository/Settings.vue

```diff
@@ -0,0 +1,134 @@
+          <set-authorization-role-form
+            ref="setAuthorizationRoleForm"
+            :repository-uuid="repository.uuid"
+            @roleSetted="onRoleSetted()" />
+          <authorizations-list
+            ref="authorizationsList"
+            :repository-uuid="repository.uuid"
+            @edit="onEditRole($event)" />
+          <h1 class="bh-title-2">Authorization Requests</h1>
+          <authorization-requests-list
+            :repository-uuid="repository.uuid"
+            @review="onReviewAuthorizationRequest()" />
+        </div>
+      </div>
+      <div
+        v-else-if="authenticated"
+        class="bh-notification">
+        <div class="bh-notification bh-notification--warning">
+          You can not edit this repository
+        </div>
+      </div>
+    </div>
+    <div
+      v-else
+      class="repository-settings__login-section bh-notification">
+      <div class="bh-notification bh-notification--info">
+        Sign in to your account to edit this repository.
+      </div>
+      <login-form hide-forgot-password />
+    </div>
+  </repository-view-base>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import RepositoryViewBase from '@/components/repository/RepositoryViewBase';
+import RepositoryBase from './Base';
+import EditProfileForm from '@/components-v1/user/EditProfileForm';
+import EditRepositoryForm from '@/components-v1/repository/EditRepositoryForm';
+import SetAuthorizationRoleForm from '@/components-v1/repository/SetAuthorizationRoleForm';
+import AuthorizationsList from '@/components-v1/repository/AuthorizationsList';
+import AuthorizationRequestsList from '@/components-v1/repository/AuthorizationRequestsList';
+import LoginForm from '@/components-v1/auth/LoginForm';
+
+
+export default {
+  name: 'RepositorySettings',
+  components: {
+    RepositoryViewBase,
+    EditProfileForm,
+    EditRepositoryForm,
+    SetAuthorizationRoleForm,
+    AuthorizationsList,
+    AuthorizationRequestsList,
+    LoginForm,
+  },
+  extends: RepositoryBase,
+  computed: {
+    ...mapGetters([
+      'authenticated',
+    ]),
+  },
+  methods: {
+    getEditInitialData() {
+      const {
+        name,
+        slug,
+        language,
+        description,
+        is_private: isPrivate,
+      } = this.repository;
+      return {
+        name,
+        slug,
+        language,
+        categories: this.repository.categories.map(
+          ({ id, name: n }) => ({ value: id, display_name: n }),
+        ),
+        description,
+        is_private: isPrivate,
+      };
+    },
+    onEdited() {
+      this.updateRepository();
+      this.editModalOpen = false;
+      this.$toast.open({
+        message: 'Repository edited!',
+        type: 'is-success',
+      });
+    },
+    onRoleSetted() {
+      this.$refs.authorizationsList.updateAuthorizations();
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+  .repository-settings {
+    padding: -1rem;
```

Comment:
Use margin, no padding to create grid. In this case just use `margin: 1rem 0;` in children elements.